config/database.yml

```diff
@@ -23,21 +23,21 @@ default: &default
 
 development:
   <<: *default
-  database: nuansacp_development
```

Comment:
Please don't change this configuration. By convention, we use `nuansacp_development` for development environment database, `nuansacp` for production, and `nuansacp_test` for test environment.

Reply:
For more information about this, you can search more about differentiating rails environment.